en/config_heli/README.md

```diff
@@ -0,0 +1,87 @@
+# Helicopter Configuration
+
+This section contains topics related to helicopter configuration and tuning.
+
+:::note
+Helicopters are less well-supported than other types of vehicles.
+We would welcome your [contribution](../contribute/README.md) of new features, new frame configurations, or other improvements.
+:::
+
+## Supported Configurations
+
+Supported helicopter configurations:
+- Single main rotor with any type of swash-plate
+- ESC tail rotor
```

Comment:
This reads as two configurations - a single main rotor and possibly a single main rotor + esc tail rotor: [Generic Helicopter (Tail ESC)](https://docs.px4.io/main/en/airframes/airframe_reference.html#copter_helicopter_generic_helicopter_(tail_esc)).

Given you can't indicate either of these are missing, do you mean:

```suggestion
Supported helicopter configurations:
- Single main rotor with any type of swash-plate and an ESC tail rotor.
```